CooperationHullMainSite/DeploymentScripts/parameters.json

```diff
@@ -0,0 +1,12 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "sites_CooperationHullMainSitev1_name": {
+            "value": null
```

Comment:
I have no idea how any of this works. Do these not need values?

Reply:
I'm not 100% sure how it all works myself but it look s about the same as the work ones.
If it goes wrong can always revert to the manual process